README.md

```diff
@@ -69,7 +69,9 @@ Parameters for the CLI tool:
 | aggregates           | list    | list of geographies for which to calculate predictions beyond the original `postal_code`, `county_fips`, `district`, `county_classification` |
 | pi_method            | string  | method for constructing prediction intervals (`nonparametric` or `gaussian`) |
 | model_parameters     | dict    | dictionary of model specific parameters e.g. `--model_parameters='{"robust":True}'` |
-| called_contests      | dict    | a dictionary of called contests. specific to Bootstrap model for now. e.g. `--called_contests='{"VA": -1}'` |
+| lhs_called_contests  | list    | a list of states called for lhs party (ie. ones where prediction > 0) |
+| rhs_called_contests  | list    | a list of states called for rhs party (ie. ones where prediction < 0) |
+| stop_model_call      | list    | a list of states for which we should override the model call |
```

Comment:
Thank you so much for making these changes!  Exciting stuff 😄 🎉 

One question I have, and I'm sure this is in the code and I'm misunderstanding it 😞  Why does it matter if the called contest is for the LHS or the RHS candidate/party? 🤔

Reply:
because we need to assign the state to that party

Reply:
😅 ok I went back through the code and the comments and I think I get it.  We use the distribution of errors on the predictions to generate aggregate intervals, but if we already have a call for a given contest, that's going to impact that distribution because the amount of error for that contest depends on for whom the contest was called.  So if we have a predicted margin for a contest like +5, but the actual call came in and was like -5, that matters because our error there should be larger than if the actual call came in and was like +3.  Do I have that even remotely right? 😅

Reply:
yes! but even beyond the aggregate prediction this matters. Think about a one contest case (e.g. VA governor), and our prediction is D+3 and our interval runs R+1 to D+7 and then we get a race call. We want to force the model to not show R+1 as a lower bound, instead we want our interval to run from 0 (or technically D+0.25 to D+7). If we are in that situation and we get an GOP racecall (obv hopefully unlikely our model is that wrong) we force the model to go from R+1 to R+0.25 (with the point prediction being force to R+0.25 also). 

Conversely, if our model is saying the confidence interval is going from D+1 to D+7 but think our model is being unreasonably bullish for democrats, if we set `stop_race_call` to true for that state then we force the lower interval to go to R+0.25. 

Does that generally make sense?

Reply:
Yes!  That's awesome!  Yay!! 😄 🎉